### ISSUE TITLE
Deprecate notice

### ISSUE THREAD

**Issue by vbrozik** [ActiveState/appdirs#188]
Could you please add the [Deprecate notice](https://github.com/ActiveState/appdirs/commit/8734277956c1df3b85385e6b308e954910533884) to:

* `setup.py` - to be shown at the time of installation
* [PyPI](https://pypi.org/project/appdirs/) - to the description
* at PyPI (in the package metadata) change the Development Status to "Inactive"

Let me know if I should make a pull request.

**Comment by sellout** [comment]
Indeed, I got here from a project that uses appdirs, and commented on an issue not realizing that it’s not developed any more.

You can [mark the repo as “archived” on GitHub](https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories), which is very visible (complementary to the suggestions @vbrozik made).

**Comment by yuwash** [comment]
Now I see that platformdirs is being recommended as replacement. It seems to indeed offer virtually the same interface, but are there any gotchas? I don’t see any information about migrating from appdirs [on their documentation](https://platformdirs.readthedocs.io/en/latest/). I was initially thinking it’s a completely different and incompatible approach as there’s no mention of appdirs at all.
